environment.go

```diff
@@ -229,6 +233,8 @@ type envState struct {
 	roleManager        gimlet.RoleManager
 	userManager        gimlet.UserManager
 	userManagerInfo    UserManagerInfo
+	// ShutdownSequenceStarted is true iff the shutdown sequence has been started
```

Comment:
nit: This comment is capitalized, while the variable isn't, and also seems redundant, since the variable is self-explanatory.